app/assets/javascripts/discourse/app/templates/about.hbs

```diff
@@ -18,12 +18,12 @@
                 "faq"
               }}</LinkTo></li>
         {{/if}}
-        {{#if (gt this.siteSettings.tos_topic_id 0)}}
+        {{#if this.site.tos_url}}
           <li class="nav-item-tos"><LinkTo @route="tos">{{i18n
```

Comment:
Should those two links (tos and privacy) use the URL instead?

Reply:
That would be ideal, but I am not sure how that interacts with Ember logic. The `tos_url` can be a) a local link to `/tos` or b) a link to where ToS is stored. At this moment, it routes to `/tos` and then that route handler redirects the user to the correct link (if necessary, case b).